SECRUITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+The Defence Digital Digital Standards Policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability, please submit your report to us using [here](#)
+In your report please include details of:
+
+- The website, IP or page where the vulnerability can be observed.
+- A brief description of the type of vulnerability, for example; "XSS vulnerability".
+- Steps to reproduce. These should be a benign, non-destructive, proof of concept. This helps to ensure that the report can be triaged quickly and accurately. It also reduces the likelihood of duplicate reports, or malicious exploitation of some vulnerabilities, such as sub-domain takeovers.
+
+Vulnerability reporting guidelines
+
+- Please do not share the vulnerability information beyond the owner and us, without express consent from the owner
+- Vulnerabilities reported to the HackerOne platform can be submitted without the need to create a HackerOne account. However, if you wish to be updated you should create an account
+- To submit your report, you will need to agree to the HackerOne Terms and Conditions and acknowledge that you have read their Privacy Policy and Disclosure Guidelines
+- Once you have submitted the report, it will be assessed by NCC Group within five working days, and forwarded to the affected owners as soon as possible.
```

Comment:
This is subject to change depending on which reporting method we choose to go with.